examples/hotrod/pkg/tracing/rpcmetrics/observer_test.go

```diff
@@ -106,39 +104,27 @@ func TestObserver(t *testing.T) {
 
 func TestTags(t *testing.T) {
 	type tagTestCase struct {
-		key     string
-		variant string
-		value   interface{}
+		attr    attribute.KeyValue
 		metrics []u.ExpectedMetric
 	}
-
 	testCases := []tagTestCase{
-		{key: "something", value: 42, metrics: []u.ExpectedMetric{
+		{attr: attribute.Key("Something").Int(42), metrics: []u.ExpectedMetric{
 			{Name: "requests", Value: 1, Tags: tags("error", "false")},
 		}},
-		{key: "error", value: true, metrics: []u.ExpectedMetric{
+		{attr: attribute.Key("Error").Bool(true), metrics: []u.ExpectedMetric{
 			{Name: "requests", Value: 1, Tags: tags("error", "true")},
 		}},
-		// OTEL bridge does not interpret string "true" as error status
-		// {key: "error", value: "true", variant: "string", metrics: []u.ExpectedMetric{
-		// 	{Name: "requests", Value: 1, Tags: tags("error", "true")},
-		// }},
 	}
 
 	for i := 200; i <= 500; i += 100 {
 		status_codes := []struct {
-			value   interface{}
-			variant string
+			value int64
 		}{
-			{value: i},
-			{value: uint16(i), variant: "uint16"},
-			{value: fmt.Sprintf("%d", i), variant: "string"},
+			{value: int64(i)},
 		}
 		for _, v := range status_codes {
```

Comment:
there is only one variant, no need to loop through it. Just use `i` as the value of the attribute.